docs/api/sandbox-option.md

```diff
@@ -1,9 +1,9 @@
 # `sandbox` Option
 
-> Create a browser window with renderer that can run inside chromium OS sandbox.
+> Create a browser window with a renderer that runs inside the Chromium OS sandbox.
```

Comment:
> As an aside, I do wonder if there is much value in using sandbox: true without --enable-sandbox?

As far as I can see, there are 3 use cases for `sandbox` option without `--enable-sandbox`:

- It is the only way to get native `window.open` behavior(This may change with #8963 though).
- It stops electron from creating a node.js runtime in the renderer, which is good from a security POV since it potentially reduces attack surface.
- For linux(and possibly windows too) you can't obtain crash dumps if `--enable-sandbox` is used because breakpad requires filesystem access to write the dump file. For now the user has to remove `--enable-sandbox` in order to debug crashes, though this may change in the future if electron starts using crashpad for all platforms. (BTW, feel free to include this information in the doc 😄 )

> Perhaps the property should be renamed sandbox-compatible: true?

Considering some people are already using the option, it is better keep the name for now. Maybe in electron 2.0.